### DISCUSSION TITLE
Which permissions are needed to cancel a github actions run

### DISCUSSION THREAD

**Discussion by david-gang** [community/community#26064] [Actions]
<p>Hi all,</p>
<p>Which permissions are needed to be able to cancel a github actions workflow run?<br>
I see that permissions are needed for such things.</p><aside class="quote" data-post="8" data-topic="17888">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/can-not-cancel-my-job-the-cancel-button-missing/17888/8">can not cancel my job, the "cancel" button missing</a> <a class="badge-wrapper  bullet" href="/c/code-to-cloud/github-actions/41"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #2188ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Discussion and questions about GitHub Actions—automation for all of your development workflows, now with world-class CI/CD.">GitHub Actions</span></a>
  </div>
  <blockquote>
    I can see the “cancel button” now. This is due to account permission problem. 
But the job alwyas hangs there after executing a couple of intel sde64 commands. 
Any idea why ?
  </blockquote>
</aside>

<p>Thanks,</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/david-gang">@david-gang</a>,</p>
<p>To cancel the workflow runs in a repository, you should be a <strong>Collaborator</strong> of the repository:</p>
<ol>
<li>
<p>If in an organization repository (see “<a href="https://docs.github.com/en/github/getting-started-with-github/types-of-github-accounts#organization-accounts" rel="nofollow noopener"><strong>Organization accounts</strong></a>”), you should be the ‘<strong>Write</strong>’ role at least.<br>
If your role is ‘<strong>Read</strong>’ or ‘<strong>Triage</strong>’, you can’t cancel workflow runs. You need to contact the owners or administrators to upgrade your role.<br>
</p>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/aca237c6cadfbb1870692f132db5e23ee922e7be.png" data-download-href="/uploads/short-url/oDbFR92SNDCbyKaxw3xfBDY5to2.png?dl=1" title="roles" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181095046-a32f9a00-95ca-4d12-9e5a-57cc230744d3.png" alt="roles" data-base62-sha1="oDbFR92SNDCbyKaxw3xfBDY5to2" width="353" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/aca237c6cadfbb1870692f132db5e23ee922e7be_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">roles</span><span class="informations">370×524 19.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
</li>
<li>
<p>If in an user repository (see “<a href="https://docs.github.com/en/github/getting-started-with-github/types-of-github-accounts#personal-user-accounts" rel="nofollow noopener"><strong>Personal user accounts</strong></a>”), once you have been invited as a <strong>Collaborator</strong> into the repository, you have the <strong>Read &amp; Write</strong> permissions.</p>
</li>
</ol>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/david-gang">@david-gang</a> ,</p>
<p>As I mentioned above, to cancel the workflow runs in a repository, you need to be a <strong>Collaborator</strong> with the <strong>Write</strong> permission in the repository.</p>
<p>How are thing going? Is my explanation helpful to you?<br>
If you have any other question about this topic, feel free to tell us.</p>

**Comment by david-gang** [comment]
<p>Thanks. This answer is good</p>
